src/libraries/System.Private.CoreLib/src/Internal/IO/File.cs

```diff
@@ -18,7 +18,7 @@ internal static partial class File
         // given by the specified path exists; otherwise, the result is
         // false.  Note that if path describes a directory,
         // Exists will return true.
-        public static bool Exists(string? path)
+        public static bool Exists([NotNullWhen(true)] string? path)
```

Comment:
You'll need to add a `using System.Diagnostics.CodeAnalysis;` to the top of the file.

Reply:
Done.